src/result.js

```diff
@@ -3,6 +3,9 @@
 const SimpleDOM = require('simple-dom');
 const HTMLSerializer = new SimpleDOM.HTMLSerializer(SimpleDOM.voidMap);
 
+const SHOEBOX_TAG_PATTERN = '<script type="fastboot/shoebox"';
+const HTML_HEAD_REGEX = /^([\s\S]*<\/head>)([\s\S]*)/;
```

Comment:
@marcoow it does not support body attributes recently added in #171

Reply:
I'm not sure what you mean - this RegExp does not make any assumptions on how the `<body>` looks and works perfectly fine with a tag like `<body data-foo="my value">`.

Reply:
@marcoow ah sorry for confusion, my bad - it says `head>` and I was reading `body>` 🤦